src/backend/joanie/core/factories.py

```diff
@@ -703,10 +707,54 @@ class Meta:
         OrderFactory,
         product__type=enums.PRODUCT_TYPE_CREDENTIAL,
     )
+    signed_on = None
```

Comment:
I wonder if `None` is not the default value if any value is provided to the factory.

Reply:
If you mean that this could be removed, it didn't work.
Test would fail with: 
```
AttributeError: The parameter 'signed_on' is unknown.
```